packages/react-vapor/src/components/editor/JSONEditorUtils.ts

```diff
@@ -0,0 +1,12 @@
+const validateValue = (jsonValue: string) => {
+    try {
+        JSON.parse(jsonValue);
+        return true;
+    } catch (error) {
```

Comment:
is it usefull to pass the `error` if you don't use it ?

Reply:
Oops, I'll remove it!